external/header.php

```diff
@@ -73,7 +73,7 @@
 }
 unset($dir);
 
-if (!extension_loaded('mongo') && Xhgui_Config::read('save.handler') === 'mongodb') {
+if ((!extension_loaded('mongo') || !extension_loaded('mongodb')) && Xhgui_Config::read('save.handler') === 'mongodb') {
```

Comment:
This:

    !extension_loaded('mongo') || !extension_loaded('mongodb')

Should be:

    !extension_loaded('mongo') && !extension_loaded('mongodb')